playbooks/prepare_host.yaml

```diff
@@ -69,3 +69,39 @@
     user:
       name: stack
       groups: wheel
+  - name: Prepare host for Ceph
+    when: ceph_enabled
+    block:
+    - name: Make sure we have losetup installed/latest
+      package:
+        name:
+          - util-linux
+          - lvm2
+        state: latest
+    - name: Use dd and losetup to create the loop devices
+      shell: |
+        dd if=/dev/zero of=/var/lib/ceph-osd.img bs=1 count=0 seek={{ ceph_loop_device_size }}G
+        losetup /dev/loop3 /var/lib/ceph-osd.img
+        pvcreate /dev/loop3
+        vgcreate vg2 /dev/loop3
+        lvcreate -n data-lv2 --size {{ ceph_loop_device_size * 96 / 100 }}G vg2
+        lvcreate -n db-lv2 --size {{ (ceph_loop_device_size * 4 / 100) - 0.1 }}G vg2
+    - name: Create /etc/systemd/system/ceph-osd-losetup.service
+      copy:
+        dest: /etc/systemd/system/ceph-osd-losetup.service
+        content: |
+          [Unit]
+          Description=Ceph OSD losetup
+          After=syslog.target
+          [Service]
+          Type=oneshot
+          ExecStart=/bin/bash -c '/sbin/losetup /dev/loop3 || \
+          /sbin/losetup /dev/loop3 /var/lib/ceph-osd.img ; partprobe /dev/loop3'
```

Comment:
I think the partprobe is redundant here because there's no partition table on it.

Reply:
I'll leave it here, I stole it from tripleo/ceph CI, I'm sure there is a reason behind this, maybe it's for the LVM partitions?